src/containers/List/List.tsx

```diff
@@ -237,7 +236,7 @@ export const List = ({
         orderWith: tableVals.sortCol,
       },
     };
-  }, [searchVal, tableVals, filters]);
+  }, [searchVal, tableVals, filters, filtersTag]);
```

Comment:
```suggestion
  }, [searchVal, tableVals, filters]);
```
It can be added in the filters object only.